profiles/source_profile.go

```diff
@@ -446,13 +446,19 @@ type DatastreamConnProfile struct {
 }
 
 type DataflowConfig struct {
-	Location            string `json:"location"`
-	Network             string `json:"network"`
-	Subnetwork          string `json:"subnetwork"`
-	HostProjectId       string `json:"hostProjectId"`
-	MaxWorkers          string `json:"maxWorkers"`
-	NumWorkers          string `json:"numWorkers"`
-	ServiceAccountEmail string `json:"serviceAccountEmail"`
+	ProjectId            string `json:"projectId"`
+	Location             string `json:"location"`
+	Network              string `json:"network"`
+	Subnetwork           string `json:"subnetwork"`
+	VpcHostProjectId     string `json:"vpcHostProjectId"`
```

Comment:
Can renaming this lead to backward compatibility issues in the CLI?

Reply:
If someone is using an old streamingcfg/sharded config and configuring vpc, then yes this could cause an issue. But this renaming is important as hostprojectid is only relevant to vpc.

Reply:
Done